docs_source_files/content/grid/grid_4/components_of_a_grid.ja.md

```diff
@@ -3,6 +3,12 @@ title: "グリッドのコンポーネント"
 weight: 1
 ---
 
+{{% notice info %}}
```

Comment:
Hi @mookkiah ,

Could you please let me know why this is added ?

As the page is already translated, we've removed the message headers. If there is any change required in translated doc
1. Please replace the translated text with the text in english (or) entire paragraph in this case
2. Add the notice as it needs to be translated later

Reply:
It does makes sense to replace the changed section in english to help the translator to find it quickly.
I did it that way. @JS31096 please check.